data/json/items/ranged/slings.json

```
@@ -1,6 +1,6 @@
 [
   {
-    "id": "sling",
+    "id": "shepherd's_sling",
```

```suggestion
    "id": "sling",
```
That's the id the game uses not the name, if you change this without migration everyone with a save with a sling in will no longer have a sling, we avoid changing ids when it's not necessary. You want to change the name str field a few lines down